skylines/frontend/views/users.py

```diff
@@ -28,7 +28,7 @@ def index():
         .options(joinedload(User.club)) \
         .order_by(func.lower(User.name))
 
-    fields = ['id', 'name']
+    fields = ['id', 'name', 'upload_permission_club']
```

Comment:
since we only need this field for the upload list and that one is requested with the `club` argument we could put `fields.append('upload_permission_club')` in the condition below.